pkg/apis/triggers/v1alpha1/interceptor_types.go

```diff
@@ -20,10 +19,14 @@ type InterceptorInterface interface {
 // Do not generate DeepCopy(). See #827
 // +k8s:deepcopy-gen=false
 type InterceptorRequest struct {
-	// Body is the incoming HTTP event body
-	Body json.RawMessage `json:"body,omitempty"`
+	// Body is the incoming HTTP event body. We use a "string" representation of the JSON body
+	// in order to preserve the body exactly as it was sent (including spaces etc.). This is necessary
```

Comment:
Let's also mention the json.Marshal/base64 behavior - while`[]byte` can be an exact representation of a string, it's the marshal behavior that makes it unusable.

Reply:
done!